packages/create-sitecore-jss/src/templates/nextjs-multisite/src/lib/middleware/plugins/multisite.ts

```diff
@@ -32,6 +32,7 @@ class MultisitePlugin implements MiddlewarePlugin {
   }
 
   async exec(req: NextRequest, res?: NextResponse): Promise<NextResponse> {
+    // @ts-expect-error A 'Property '[INTERNALS]' is missing' error can occur in monorepo setups with peerDeps.
```

Comment:
This note is added to fix the error occurring during nextjs build. Full text:
```
Type error: Property '[INTERNALS]' is missing in type 'import("C:/Wok/jss/jss/packages/sitecore-jss-nextjs/node_modules/next/dist/server/web/spec-extension/response").NextResponse<unknown>' but required in type 'import("C:/Wok/jss/jss/samples/nextjs/node_modules/next/dist/server/web/spec-extension/response").NextResponse<unknown>'.
```

This is a very confusing error, as the dependency installed in jss-nextjs package and nextjs sample are identical (version, contents, code, all of it). This makes me believe it is a yarn monorepo issue - yarn does not install peer dependencies - so we need to install next as devDep explicitly in `sitecore-jss-nextjs` and this causes confusion (as usually there would only be one copy of next).

Reply:
Alas, adding `next` to externals does not work. It is a typescript, you will also see it when editing in vscode.
Do you know a way to somehow reuse one single package via tsconfig configuration?